packages/rancher-logging/app-readme.md

```diff
@@ -0,0 +1,14 @@
+# Rancher Logging
+
+Rancher logging is powered by the [Bonzai Logging Operator]()
+
+Rancher logging allows users to configure complex logging pipelines with a few simple resources.  
+
+[ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/) and [ClusterOutput](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) allow for collecting all logs across a cluster.
+The namespaced resources [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) and [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) allow for collecting logs from a single namespace.
```

Comment:
Should make this active voice rather than passive, e.g.

To collect logs from components deployed within a specific namespace, users can define [Flows](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) that <insert a description of what they do / what could be specified> and [Outputs](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) that <insert a description of what they do / what could be specified>.

To collect logs from components deployed across the entire cluster, users can similarly define [ClusterFlows](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/) and [ClusterOutputs](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) that store their logging configuration.